app.js

```diff
@@ -0,0 +1,103 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+
+var index = require('./routes/index');
+var users = require('./routes/users');
+
+var app = express();
+
```

Comment:
I'm sure that the generator made them like this, but you should change all of the vars here to const.